build/dockerfiles/Dockerfile

```diff
@@ -9,22 +9,16 @@
 FROM docker.io/alpine:3.11.5 AS builder
 RUN apk add --no-cache py-pip jq bash wget git skopeo && pip install yq
 
-# Registry, organization, and tag to use for base images in dockerfiles. Devfiles
-# will be rewritten during build to use these values for base images.
-ARG PATCHED_IMAGES_REG="quay.io"
-ARG PATCHED_IMAGES_ORG="eclipse"
-ARG PATCHED_IMAGES_TAG="next"
 ARG USE_DIGESTS=false
```

Comment:
in what user story will we ever want this to be true?

Reply:
Removed